models/classes/runner/QtiRunnerService.php

```diff
@@ -614,10 +614,10 @@ public function getRubrics(RunnerServiceContext $context, AssessmentItemRef $ite
      * @param string $itemRef
      * @return string
      */
-    public function getItemHref(RunnerServiceContext $context, $itemRef)
+    public function getItemHref(RunnerServiceContext $context, string $itemRef): string
```

Comment:
I would drop phpdoc annotation (across modified files) which are not adding value on top of the method signature ( especially after you've added strict typing )

Reply:
I usually prefer to keep the file consistent. That's the reason why I'm using `phpdoc` for the new methods. I'd prefer to keep them if you don't mind and perhaps we can get rid of them in a near future to avoid too much noise in this big PR. ;)